next.clayui.com/content/docs/css/components/labels.md

```diff
@@ -0,0 +1,340 @@
+---
+title: "Labels"
+description: "Labels categorize information, providing quick recognition."
+---
+
+<div class="nav-toc">
+
+- [Label Variations](#label-variations)
+- [Sizes](#sizes)
+- [Variations](#variations)
+	- [Simple Label](#simple-label)
+	- [Label Dismissible](#label-dismissible)
+	- [Label with Link](#label-with-link)
+	- [Interactive Label](#interactive-label)
+	- [Interactive Label Examples](#interactive-label-examples)
```

Comment:
We can remove this since we do not have that session here.

Reply:
Updated!